dispatch/apps/api/serializers.py

```diff
@@ -14,13 +14,46 @@ class UserSerializer(DispatchModelSerializer):
     """
     Serializes the User model.
     """
+
+    email = serializers.EmailField(required=True)
+    person_id =  serializers.IntegerField(validators=[validate_person_id])
+    password_a = serializers.CharField(
+        required=False, 
+        write_only=True, 
+        validators=[PasswordValidator(confirm_field='password_b')]
+    )
+    password_b = serializers.CharField(required=False, write_only=True)
+
     class Meta:
         model = User
         fields = (
             'id',
-            'email'
+            'email',
+            'person_id',
+            'password_a',
+            'password_b',
         )
 
+    def create(self, validated_data):
+
+        instance = User()
+        if User.objects.filter(email=validated_data['email']).exists():
```

Comment:
What happens if you try to call `instance.save()` with a duplicate email? Does it raise an error? I'm thinking it might be better to catch the error when it is raised instead of trying to validate the data beforehand